charts/tfy-llm-gateway/values.yaml

```diff
@@ -1,13 +1,29 @@
 ## @section Configuration for LLM Gateway
 
 ## @param global Truefoundry global values
-global: {}
+global:
+  ## @param global.controlPlaneURL Control plane URL
+  controlPlaneURL: "http://truefoundry-truefoundry-frontend-app.truefoundry.svc.cluster.local:5000"
```

Comment:
lets not hardcode this as we are also using this in DEPLOYED_LLM_GATEWAY_URL env - which is user facing used in code snippet